mucgpt-frontend/src/components/DepartmentTreeDropdown/DepartmentTreeDropdown.module.css

```
@@ -14,20 +14,20 @@
     font-size: 13px;
-    color: var(--onSurface);
-    border: 1px solid var(--outline);
+    color: var(--colorNeutralForeground1);
+    border: 1px solid var(--colorNeutralStroke1);
     transition: all 0.2s ease;
     font-weight: 500;
 }
 
 .selected_item:hover {
-    background: var(--outline);
+    border: 1px solidvar(--colorBrandBackground);
```

Stylelint (17.6.0)

[error] Expected "--colorBrandBackground" to be "--colorbrandbackground" (value-keyword-case)

(value-keyword-case)